scripts/prepare-pages-folder.sh

```diff
@@ -1,27 +1,39 @@
 #!/bin/bash
 
+echo 'starting gh-pages creation ...'
 VERSION=$(node -p "require('./package.json').version")
 
+# checkout current gh-pages branch
+echo 'creating gh-pages folder ...'
+mkdir gh-pages
+echo 'checking out gh-pages branch ...'
```

Comment:
Let's make this more verbose : 

```suggestion
echo "checking out https://github.com/$TRAVIS_REPO_SLUG gh-pages branch ..."
```